networking/src/p2p/peer.rs

```diff
@@ -29,7 +29,7 @@ use super::network_channel::{NetworkChannelRef, NetworkChannelTopic, PeerBootstr
 use super::stream::{EncryptedMessageReader, EncryptedMessageWriter, MessageStream, StreamError};
 
 const IO_TIMEOUT: Duration = Duration::from_secs(6);
-const READ_TIMEOUT_LONG: Duration = Duration::from_secs(30);
+const READ_TIMEOUT_LONG: Duration = Duration::from_secs(120);
```

Comment:
@adonagy 
what is original tezos timeout for this?

Reply:
Hmm, good question. Tezos sends a `get_current_head` to a peer which is not responding for 90s. In tezedge, we schedule and send a `get_current_head` message every 90s. So I set the timeout 120s here so it has  30s (120 - 90) to respond.